css/style.css

```diff
@@ -136,4 +136,10 @@ form {
   color: inherit;
   border-radius: 2px;
   cursor: pointer;
+}
+.end-form{
+ width: 200px;
+ height: 2px;
+ border: 2px solid black;
+ margin-left: 200px;
 }
```

Comment:
If I may suggest your work will look more professional if you would add a space before  open brace "{"